src/ann_solo/config.py

```diff
@@ -224,6 +224,26 @@ def __init__(self) -> None:
             '--fragment_tol_mode', type=str, choices=['Da', 'ppm'], default='ppm',
             help='Fragment mass tolerance unit. Either (options: %(choices)s)')
 
+        # Prosit
+        # Prosit batch size.
+        self._parser.add_argument(
+            '--prosit_batch_size', default=1000, type=int,
+            help='maximum number of peptides to generate predictions for '
+                 'using prosit server (default: %(default)s)')
+        # Collision energy.
+        self._parser.add_argument(
+            '--collision_energy', default=32, type=int,
```

Comment:
**suggestion:** Can we make this a list to support predicting spectra with multiple collision energies?